pkg/pocket/executor/abtest.go

```diff
@@ -147,10 +160,13 @@ func (e *Executor) abTestSelect(sql string) error {
 		err2 error
 	)
 	wg.Add(2)
-	go func() {
+	go func(sql string) {
+		if e.TiFlash {
+			sql = setIsolationEngine + sql
```

Comment:
`setIsolationEngine` set a session variable, it does not need to be executed everytime.